File: TrabajoAcademicoNet/SoftProgModelo/PagoModelo/LineaComprobantePago.cs
using PUCP.Inf30.SoftProg.Modelo.BaseModelo;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PUCP.Inf30.SoftProg.Modelo.PagoModelo
{
    public class LineaComprobantePago : Registro
    {
        public double MontoPagado { get; set; }
        public int Codigo { get; set; }
        public int Cantidad { get; set; }
        public double SubTotal { get; set; }
        public double MontoImpuesto { get; set; }
        public int IdComprobantePago { get; set; }

        public LineaComprobantePago()
        {
        }
        public LineaComprobantePago(double montoPagado, int codigo, int cantidad, double subTotal, double montoImpuesto, int idComprobantePago)
        {
            this.MontoPagado = montoPagado;
            this.Codigo = codigo;
            this.Cantidad = cantidad;
            this.SubTotal = subTotal;
            this.MontoImpuesto = montoImpuesto;
            this.IdComprobantePago = idComprobantePago;
        }
    }
}

File: Java/TrabajoAcademico/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/modelo/pago/LineaComprobantePago.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package pe.edu.pucp.inf30.softprog.modelo.pago;

import pe.edu.pucp.inf30.softprog.modelo.base.Registro;

/**
 *
 * @author Cristhian Horacio
 */
public class LineaComprobantePago extends Registro{
    private double montoPagado;
    private int codigo;
    private int cantidad;
    private double subTotal;
    private double montoImpuesto;
    private ComprobantePago comprobantePago;
    
    public LineaComprobantePago(){
        
    }

    public LineaComprobantePago(double montoPagado, int codigo, int cantidad, double subTotal, double montoImpuesto, ComprobantePago comprobantePago) {
        this.montoPagado = montoPagado;
        this.codigo = codigo;
        this.cantidad = cantidad;
        this.subTotal = subTotal;
        this.montoImpuesto = montoImpuesto;
        this.comprobantePago = comprobantePago;
    }

    public LineaComprobantePago(double montoPagado, int codigo, int cantidad, double subTotal, double montoImpuesto, ComprobantePago comprobantePago, int id, boolean activo) {
        super(id, activo);
        this.montoPagado = montoPagado;
        this.codigo = codigo;
        this.cantidad = cantidad;
        this.subTotal = subTotal;
        this.montoImpuesto = montoImpuesto;
        this.comprobantePago = comprobantePago;
    }
    
    public double getMontoPagado() {
        return montoPagado;
    }

    public void setMontoPagado(double montoPagado) {
        this.montoPagado = montoPagado;
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public int getCantidad() {
        return cantidad;
    }

    public void setCantidad(int cantidad) {
        this.cantidad = cantidad;
    }

    public double getSubTotal() {
        return subTotal;
    }

    public void setSubTotal(double subTotal) {
        this.subTotal = subTotal;
    }

    public ComprobantePago getComprobantePago() {
        return comprobantePago;
    }

    public void setComprobantePago(ComprobantePago comprobantePago) {
        this.comprobantePago = comprobantePago;
    }

    public double getMontoImpuesto() {
        return montoImpuesto;
    }

    public void setMontoImpuesto(double montoImpuesto) {
        this.montoImpuesto = montoImpuesto;
    }
    
    
}
